src/client/components/pages/Courses/CourseInstanceTableBody.tsx

```diff
@@ -0,0 +1,99 @@
+import { CellLayout } from 'client/components/general';
+import { COURSE_TABLE_COLUMN, getAreaColor, TERM } from 'common/constants';
+import CourseInstanceResponseDTO from 'common/dto/courses/CourseInstanceResponse';
+import {
+  TableBody, TableRow, TableRowHeadingCell, VALIGN, TableCell,
+} from 'mark-one';
+import React, { FunctionComponent, ReactElement, Ref } from 'react';
+import { CourseInstanceListColumn } from './tableFields';
+
+interface CourseInstanceTableBodyProps {
+  /**
+   * The list of courses to be shown in the table
+   */
+  courseList: CourseInstanceResponseDTO[];
+  /**
+   * The data to display
```

Comment:
This doc block isn't quite right. `CourseInstanceListColumn` represents one table column, while the doc block and param suggest that it has something to do with the table body data.

Reply:
This is the list of course instances to be displayed being passed down to the table body component after the courses are filtered by the column filters.

Reply:
Oh, I see. I was getting `CourseInstanceListColumn[]` confused with `COURSE_TABLE_COLUMN[]` and thinking that `tableData` was an array of columns. I understand now.